csrc/serde/utils.cpp

```diff
@@ -122,59 +123,6 @@ PrimDataType mapToNvfuserDtype(serde::DataType t) {
   return PrimDataType::Null;
 }
 
-::flatbuffers::Offset<serde::Scalar> serializeScalar(
```

Comment:
Moved `serializeScalar` and `parsePolymorphicValue` to polymorphic_value_serde.cpp